prep-javascript-arrays/main.js

```diff
@@ -0,0 +1,18 @@
+const languages = ['html', 'css', 'javascipt', 'react', 'node', 'postgres']
+;
+console.log(languages);
+
+var firstElement = 'html'
```

Comment:
Hi Edwin, you can access the first element of the array by its index 
```js
var firstElement = languages[0];
```

Reply:
![image](https://user-images.githubusercontent.com/90667339/135184632-6991d41c-3ff0-49ab-bd51-a0aa19418e6a.png)